### PR TITLE
DO-NOT-MERGE Invalid owner in OWNERS test

### PR DESCRIPTION
**Pull request by yt3liu**
DO NOT MERGE

Testing verify-owners plugin to see if it is working as expected. 

Fixes #

### CODE REVIEW

**Comment by adrcunha** [comment]
Just double-checking: did you deploy the new Prow plugin config?

**Comment by yt3liu** [comment]
> Just double-checking: did you deploy the new Prow plugin config?

Yes, I did deploy the plugin. I verified the 'verify-owners' plugin shows up in the plugins config on https://pantheon.corp.google.com before creating the PR.

File: OWNERS [hunk 1]
```diff
@@ -1,10 +1,3 @@
 # The OWNERS file is used by prow to automatically merge approved PRs.
```
[the remark this answers is not in the document]
Comment (yt3liu) [reply OWNERS]: Updated OWNERS file to make it valid again.

**Pull request closed without merging**